colemak/colemak_mapping.txt

```diff
@@ -0,0 +1,10 @@
+finger1 = [49, 113, 97, 122, 126, 96, 33, 81, 65, 90];
```

Comment:
seems like you haven't changed mappings. https://colemak.com/wiki/images/e/ef/Colemak_fingers.png

Reply:
Hello, sorry for that i will send you the another pull request with the changes and the layout for the qwertz keyboard in the next couple of hours